libs/ledger-live-common/src/bridge/generic-alpaca/utils.ts

```diff
@@ -177,7 +177,23 @@ export const buildOptimisticOperation = (
   transaction: TransactionCommon,
   sequenceNumber?: number,
 ): Operation => {
-  const type = transaction["mode"] === "changeTrust" ? "OPT_IN" : "OUT";
+  let type: OperationType;
+  switch (transaction["mode"]) {
```

Comment:
```suggestion
  switch (transaction.mode) {
```
Can we take this opportunity to fix the type of the transaction parameter above ? That is
```ts
transaction: GenericTransaction
```
instead of
```ts
transaction: TransactionCommon
```